CATALOG.md

```diff
@@ -35,3 +35,71 @@ nativefier https://www.udemy.com/
 ### Notes
 
 Most videos will work, but to be sure everything works you are better off using the `--widevine` version AND signing the app afterwards. See this post: [#1147 (comment)](https://github.com/nativefier/nativefier/issues/1147#issuecomment-828750362)
+
+
+## Spotify
+
+```
+nativefier https://open.spotify.com/
+--name "Spotify" 
+--widevine 
+-u "<useragent your browser uses>"
+--inject spotify.js
+--inject spotify.css
+--icon icon.
+```
+
+### Notes
+
+- [Inject](https://github.com/nativefier/nativefier/blob/master/docs/api.md#inject) the following javascript as `spotify.js` to prevent "Unsupported Browser" messages.
+```javascript
+function dontShowBrowserNoticePage() {
+    const browserNotice = document.getElementById('browser-support-notice');
+    console.log({ browserNotice })
+    if (typeof browserNotice !== "undefined" && browserNotice !== null) {
+        document.getElementsByTagName('body')[0].style.display = 'none';
+    }
```

Comment:
1. No need to be so strict with checking
2. If we found the browserNotice by id, why not accessing it (and removing it) by id? It will be more reliable. Or am I missing something and `document.getElementsByTagName('body')[0]` is something else? If it is, please comment what it is.

```suggestion
    if (browserNotice) {
        browserNotice.style.display = 'none';
    }
```

Reply:
This was my introduction. The reason being is when it displays the browser notice, it's not just the notice, but the entire page is focused on not allowing you to proceed. So in this instance we hide the body element (so nothing shows) until the JS can delete the service worker and reload (which will actually load the player).

Reply:
> This was my introduction. The reason being is when it displays the browser notice, it's not just the notice, but the entire page is focused on not allowing you to proceed. So in this instance we hide the body element (so nothing shows) until the JS can delete the service worker and reload (which will actually load the player).

@TheCleric okay, thanks. In this case,

1. Minimally, a comment please
2. If possible, an extra check (asserting that `document.getElementsByTagName('body')[0]` is what we expect it to be) would be neat

Reply:
> @TheCleric okay, thanks. In this case,
> 
> 
> 
> 1. Minimally, a comment please


Will do.

> 2. If possible, an extra check (asserting that `document.getElementsByTagName('body')[0]` is what we expect it to be) would be neat. I understand it may not be easy, in this case okay for a simple comment, and if it breaks it breaks.

It's the entire HTML page's body so I can't imagine it wouldn't be there, but I can add a check if you want to be extra careful.

Reply:
> It's the entire HTML page's body so I can't imagine it wouldn't be there, but I can add a check if you want to be extra careful.

@TheCleric duuuuuuuuuuuuuh sorry, misread :rofl: . We're hiding the whole body? Okay, I don't understand what's going on, but I trust you it works. Okay for the current code + a comment.